third_party/move/move-core/types/src/vm_status.rs

```diff
@@ -728,6 +729,11 @@ pub enum StatusCode {
     RESERVED_VERIFICATION_ERROR_3 = 1129,
     RESERVED_VERIFICATION_ERROR_4 = 1130,
     RESERVED_VERIFICATION_ERROR_5 = 1131,
+    // Verification errors related to automation transaction
+    // Automation Transaction payload validation failed.
+    INVALID_AUTOMATION_PAYLOAD = 1132,
+    INVALID_AUTOMATION_PAYLOAD_ARGUMENTS = 1133,
+    INVALID_AUTOMATION_INNER_PAYLOAD = 1134,
```

Comment:
The lines above mentions `RESERVED_VERIFICATION_ERROR_X`, does it mean that this error codes are also referenced/provisioned somewhere else? In which case these new codes would have to be integrated there as well.

Reply:
I have checked they are not referenced in any other place.